pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -372,6 +392,18 @@ func (p *Provisioner) initializeDeleteOrRevoke(ob *bktv1alpha1.ObjectBucket) err
 	if err != nil {
 		return errors.Wrapf(err, "failed to set CA cert for the OB %q to connect with object store %q via TLS", ob.Name, p.objectStoreName)
 	}
+
+	// Set admin ops api client
+	err = p.setAdminOpsAPIClient()
```

Comment:
This block looks the same as above, can it be factored to a helper method? Or maybe `setAdminOpsAPIClient()` just needs to do the special error handling?

Reply:
I'm not sure there is much to factor in here, saving one line, it's already the top-level caller too. Also, yes they are the same just like a bunch of code since both are called under different circomstances.